doc/api/fs.md

```diff
@@ -2400,6 +2400,9 @@ a `Promise` for an `Object` with `bytesRead` and `buffer` properties.
 <!-- YAML
 added: v0.1.8
 changes:
+  - version: v10.10.0
+    pr-url: https://github.com/nodejs/node/pull/22020
+    description: Added new option `withFileTypes`.
```

Comment:
Nit: maybe `` New option `withFileTypes` was added ``? The most similar entries seem to follow this pattern in this doc.